WPMedia/fusion-news-theme-blocks
# Tmedia 763 chains

## Description

This updates the single, double, triple and quad chains to the new styling system.  It attempts to match as close as possible the styles expressed in this sample page:  https://corecomponents-the-gazette-prod.cdn.arcpublishing.com/chain-blocks/?_website=the-gazette.  The feature pack PR can be found here: ~~https://github.com/WPMedia/arc-themes-feature-pack/pull/245~~
https://github.com/WPMedia/arc-themes-feature-pack/pull/249

## Jira Ticket

- https://arcpublishing.atlassian.net/browse/TMEDIA-720

## Acceptance Criteria

Acceptance Criteria:

Chains blocks are migrated to the new architecture, including:

Single Chain

Includes Chain Configuration ‘Heading’ field

Double Chain

Includes Chain Configuration ‘Heading’ field, ‘Number of blocks in Column 1’ with help text

Triple Chain

Includes Chain Configuration ‘Heading’ field, ‘Number of blocks in Column 1’ with help text, ‘Number of blocks in Column 2’ with help text

Quad Chain

Includes Chain Configuration ‘Heading’ field, ‘Number of blocks in Column 1’ with help text, ‘Number of blocks in Column 2’ with help text, ‘Number of blocks in Column 3’ with help text

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout tmedia-763-chains`
2. Run storybook `npm run storybook`
3. Compare the 4 chains to an older storybook build under the news theme: https://5f91e4721ccaba0022a10782-ovouhewamv.chromatic.com/?path=/story/chains-triple--column-one-only.  Note the only difference you should see is that I have spacing under the header to match what I see in https://corecomponents-the-gazette-prod.cdn.arcpublishing.com/chain-blocks/?_website=the-gazette


## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
